README.md

```diff
@@ -176,7 +176,7 @@ main() {
 
     // by default set [rgba] and [xyza] conventions
 
-    // add new          red   green  blue  alpha
+    // add           red   green  blue  alpha
```

Comment:
now labels of components has offset of three chars. please put three spaces back instead of word `new` that was remove

Reply:
Changed back to old version.